src/rm.js

```diff
@@ -149,6 +149,11 @@ function _rm(options, files) {
     if (stats.isDirectory() && options.recursive) {
       rmdirSyncRecursive(file, options.force);
     }
+
+    // Path is a symbolic link
+    if (stats.isSymbolicLink()) {
```

Comment:
Also, we have lots of code duplication here that should probably get cleaned up. We call `common.unlinkSync()` in two different branches for `isFile`, and we should probably move the `isDirectory` branches into one:

```js
if (isFile) {
  if (options.force || isWriteable()) {
    // remove
  } else {
    // error
  }
} else if (isDirectory()) {
  if (options.recursive) {
     // remove
  } else {
     // error
  }
} else if (isLink()) {
  // remove
}
```